### ISSUE TITLE
`Block` fragment is a little confusing

### ISSUE THREAD

**Issue by mwildehahn** [speee/jsx-slack#11]
The `Block` fragment that contains all of the blocks is a little misleading since it isn't a single block but an array of blocks that you provide as children.

ie currently you have:

```
<Block>
   <Section ...>
   <Section ...>
</Block>
```

Is there a reason this isn't just:

```
<Blocks>
   <Section ...>
   <Section ...>
</Blocks>
```
?

**Comment by yhatt** [comment]
Thanks for catch. We will add `<Blocks>` as the main root container.

For compatibility, deprecated `<Block>` would work just as an alias to `<Blocks>`.

**Comment by yhatt** [comment]
Updated to use `<Blocks>` instead of `<Block>` in #12, and released as [v0.4.1](https://github.com/speee/jsx-slack/releases/tag/v0.4.1).
